feature(nodeview): add tooltip

burgerni10 opened the pull request.

burgerni10 commented:
This is a draft to fix the issue #1298 

For this to be finalised, the dataSourceId must be unique and css compatible (no space for example). I suggest we wait for the issue #1191 to be done before going back to this one.

jfhenon reviewed (changes requested):
check the use of random in South/North loops.

burgerni10 commented:
I moved the targeted id of each tooltip to the main div instead to have it in the div having the applicationId/dataSourceId : it is more reactive this way when the mouse comes hover the node.

There was a small error in the css with the shadow.

Lastly, I rebased the branch and commits to have a clean working tree.

@franckfotso1 check the corrections when you will have time
